### DISCUSSION TITLE
Hotkey doesn't work as expected

### DISCUSSION THREAD

**Discussion by Alberth-lz** [Belphemur/SoundSwitch#833] [Q&A] [answered]
First, I'd like to say thanks for this app. The problem I've been having is that hotkey doesn't work unless I open manually the app. After that, it works perfect but it's necessary every time I turn on the pc. 
If someone could help me with this will be awesome and thanks in advance

**Comment by weltern754** [comment]
I got exactly the same problem with windows 11.  I had soundswitch at startup to run, but the hotkeys doesn't work. I close the app and reopen manually and it works fine.

**Reply by Belphemur** [reply]
@Alberth-lz  & @weltern754 Can you double-check you're on the latest version ?

**Reply by weltern754** [reply]
for me yes, and a clean reinstall...

**Reply by weltern754** [reply]
Hello  @Belphemur, sorry for interupt your precious time, I am hoping for an  for this, and to know if  you found something? have a nice day

**Comment by glain42** [comment]
I can confirm that behavior. Completely new installation. After installing the app the hotkey works just fine, but after I reboot the app I have to to open the app (and toggle the hotkey option) to get it to stick. I've remapped the hotkey (ALT+CTRL-F12) for output devices.

**Reply by harmi97** [reply]
v6.2.4.0 - I have same issue (win10)

**Comment by SGY118** [comment]
any solution? I have exactly the same issue. I couldn't find it in the task manager startup. So I have to reopen the app to make the hot key work.

**Comment by Belphemur** [comment, marked as answer]
This should be fixed in v6.4.3

**Reply by weltern754** [reply]
Oh wow ! awesome ! I will try and tell you soon !

**Reply by weltern754** [reply]
your the best ! :D it worked! Thank you thank you i like so much soundswitch even more
